### PR TITLE
Dockernize

### PR DESCRIPTION
**Pull request by kokuyouwind**
# Feature

Add docker settings. :whale:
You can run development server with below command.
```
docker-compose up
``` 
And, run production server with below.
```
docker-compose -f docker-compose.prod.yml up
```

Automated Build Image is in https://hub.docker.com/r/kokuyouwind/esaba/ .
I will turn source branch to master when this PR is merged.

### CODE REVIEW

File: README.ja.md [hunk 1]
```diff
@@ -44,8 +61,33 @@ $ vi config/config.secret.php   # tailor to your env
...
+
 ブラウザで http://localhost:8888/index_dev.php/post/:post_number へアクセス。
 
+### 本番設定のサーバー起動
+
+Dockerの場合は[DockerHubのAutomated Build](https://hub.docker.com/r/kokuyouwind/esaba/)を使用して本番設定のApacheサーバを起動できます。
+
+`docker-compose.prod.yml` を使用する場合、`config`以下は名前付きボリュームになるため、`config.secret.php`を初回のみ設定することで、以降はコンテナを作り直しても設定が保持されます。
+
+```php
```
Comment (mzp) [inline README.ja.md]: this code is not php. Use `shell`, or `bash`.

Comment (kokuyouwind) [reply README.ja.md]: whoops

File: README.md [hunk 2]
```diff
@@ -49,8 +65,32 @@ You must to issue personal access token in advance.
...
+
+```bash
+$ docker-compose up # It take a while to install composer & npm libraries in first 
+```
+
 And go to http://localhost:8888/index_dev.php/post/:post_number
 
+### Runnin in production
+
+You can run production Apache server with [DockerHub Automated Build Image](https://hub.docker.com/r/kokuyouwind/esaba/).
```
Comment (mzp) [inline README.md]: > DockerHub Automated Build Image

it is good to use docker image name, i.e. `kokuyouwind/esaba:latest`.

File: README.md [hunk 3]
```diff
@@ -28,7 +28,12 @@ esaba hosts your markdown docs on [esa.io](https://esa.io). Url like `/post/:pos
 - [Composer](https://getcomposer.org/)
 - [npm](https://www.npmjs.com/)
 
+Or
+
+- [Docker](https://www.docker.com/)
+
 ## Installation
+### Native Installation
```
Comment (mzp) [inline README.md]: native??? Use `Installation without docker`?

File: README.md [hunk 4]
```diff
@@ -49,8 +65,32 @@ You must to issue personal access token in advance.
 $ COMPOSER_PROCESS_TIMEOUT=0 composer run
 ```
 
+If you use Docker, alternatively run below command.
```
Comment (mzp) [inline README.md]: "below" is a prepositionl, use "following command".

File: README.md [hunk 5]
```diff
@@ -28,7 +28,12 @@ esaba hosts your markdown docs on [esa.io](https://esa.io). Url like `/post/:pos
 - [Composer](https://getcomposer.org/)
 - [npm](https://www.npmjs.com/)
 
+Or
+
+- [Docker](https://www.docker.com/)
```
Comment (mzp) [inline README.md:33]: Strictly speaking, this code requires not only docker, but also docker-compose.

At Linux platform, docker-compose is not contained in docker package. https://docs.docker.com/compose/install/

File: README.ja.md (hunk 1 above)
Comment (mzp) [reply README.ja.md]: You love PHP, too much.

Comment (kokuyouwind) [reply README.ja.md]: fixed https://github.com/standfirm/esaba/pull/2/commits/b3851fb9dbf3acbb08caade0c50302f22c222880

File: README.md (hunk 2 above)
Comment (kokuyouwind) [reply README.md]: fixed https://github.com/standfirm/esaba/pull/2/commits/b3851fb9dbf3acbb08caade0c50302f22c222880

File: README.md (hunk 3 above)
Comment (kokuyouwind) [reply README.md]: fixed https://github.com/standfirm/esaba/pull/2/commits/b3851fb9dbf3acbb08caade0c50302f22c222880

File: README.md (hunk 4 above)
Comment (kokuyouwind) [reply README.md]: fixed https://github.com/standfirm/esaba/pull/2/commits/b3851fb9dbf3acbb08caade0c50302f22c222880

File: README.md (hunk 5 above)
Comment (kokuyouwind) [reply README.md:33]: fixed https://github.com/standfirm/esaba/pull/2/commits/b3851fb9dbf3acbb08caade0c50302f22c222880

**Review by mzp — APPROVED** [review]

**Pull request merged**